src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int LENGTH_ARRAY = 10;
+    private int counter;
+    private Object[] key;
+    private Object[] value;
```

Comment:
```suggestion
    private Object[] keys;
    private Object[] values;
```
please rename the fields so that we didn't have the name collision